### DISCUSSION TITLE
Геопространственные данные

### DISCUSSION THREAD

**Discussion by Davud77** [totumonline/totum-ru-issues-and-discussions#124] [Неопределенное]
Доброго всем времени суток. У меня есть база данных PosyGIS, хотелось бы данные оттуда видеть и редактировать в Тотум, возможно ли такое? Может кто-то сталкивался?
![image](https://user-images.githubusercontent.com/26651156/165743197-5ad30779-baf4-42fe-b36d-f77c45d72ccc.png)

**Comment by ttm-alexeypolunin** [comment]
Только если у нее есть web-коннектор, к которому можно отправить запрос и который в JSON например отвечает.

**Reply by ttm-alexeypolunin** [reply]
Вижу, что это надстройка на Postgres — коннектор можно написать, но это надо скилл в самом PosyGIS

**Reply by ttm-alexeypolunin** [reply]
Вобщем да, написать небольшой скрипт в котором будет доступ к базе PostG и SQL-запросы. Дальше вызывать из Тотума их через `execSSH` с передачей нужный параметров и по полученному ответу что-то делать — временную таблицу заполнять, класть в простую итп.  У нас в паре проектов так коннект к базе CMS сделаны.

База PostG должна быть доступна по сети конешн.
